doc/_includes/tutorials/common/creating-a-user-account.md

```diff
@@ -19,3 +19,5 @@ When you click "Create User", an e-mail will be sent to the user (to the e-mail
 On successfully creating a new user account, you will see the user details that you entered:
 
 ![User details page.]({{ site.baseurl }}/images/tutorials/common/users/user-details-page.png)
+
+**Note**: Announcements created within the last month will be shown to new users.
```

Comment:
Tiny change: Can you change this to "**Only** announcements created within the last month..."

Reply:
Please see 717ef7e031697aafb3e797d0fc02797595551e64.